javascripts/endGame.js

```diff
@@ -0,0 +1,59 @@
+function checkWin() {
+  var winningTiles = $("[data-val=" + winningTileValue + "]");
+  if (winningTiles.length > 0) {
+    priorWin = true;
+    winnerTile(winningTiles);
+    endGame('win');
+  } else { return false;}
```

Comment:
Actually, is the `else` needed? It doesn't look like `checkWin()` needs to return a value.